.github/workflows/all-green.yml

```diff
@@ -45,4 +45,4 @@ jobs:
 # - dd-gitlab/download-serverless-artifacts : 86%
 # - dd-gitlab/download-single-step-artifacts : 88%
 # - throughput_profiler : 89%
-# - integration_tests_arm64_debugger : 89%
\ No newline at end of file
+# - integration_tests_arm64_debugger : 89%
```

Comment:
I briefly got my hopes up that this number had changed when I saw you had edited it 😂

Reply:
😅